Stratis.Bitcoin.Features.Wallet/Wallet.cs

```diff
@@ -614,16 +614,16 @@ public List<UnspentOutputReference> GetSpendableTransactions(int currentChainHei
                 // When calculating the confirmations the tip must be advanced by one.
 
                 var countFrom = currentChainHeight + 1;
-                var unspentTransactions = address.UnspentTransactions()
-                    .Where(a => countFrom - (a.BlockHeight ?? countFrom) >= confirmations).ToList();
+                var unspentTransactions = address.UnspentTransactions().Select(s =>
```

Comment:
we could incrementally improve the code style when we are touching old code ...
could we un-var this please?